lbrynet/daemon/Daemon.py

```diff
@@ -168,7 +168,8 @@ class Daemon(AuthJSONRPCServer):
     """
 
     allowed_during_startup = [
-        'daemon_stop', 'status', 'version', 'wallet_unlock'
+        'daemon_stop', 'status', 'version', 'wallet_unlock', 'commands', 'help', 'settings_get',
```

Comment:
Let's move this out to a decorator

Reply:
Actually, let's have commands with no required components be allowed during startup. Then we can get rid of this attribute.